Sources/App/Commands/TriggerBuilds.swift

```diff
@@ -448,16 +452,27 @@ func missingPairs(existing: [BuildPair]) -> Set<BuildPair> {
 func findMissingBuilds(_ database: Database,
                        packageId: Package.Id) async throws -> [BuildTriggerInfo] {
     let versions = try await Version.query(on: database)
-        .with(\.$builds)
         .filter(\.$package.$id == packageId)
         .filter(\.$latest != nil)
+        .field(Version.self, \.$id)
+        .field(Version.self, \.$packageName)
+        .field(Version.self, \.$reference)
+        .field(Version.self, \.$spiManifest)
+        .all()
+    let builds = try await Build.query(on: database)
+        .filter(\.$version.$id ~~ versions.compactMap(\.id))
+        .field(Build.self, \.$platform)
+        .field(Build.self, \.$swiftVersion)
+        .field(Build.self, \.$version.$id)
```

Comment:
This is a slightly unrelated change: we were pulling _every_ field in `Version` and `Build` to create a trigger when we only needed a handful. This trims the list considerably.

Reply:
Note that we were running two queries before as well. `.with(\.$builds)` runs an additional query just like we do here and fetches all the siblings of the parent relation. This query style doesn't support `.field` modifiers so we have to run the queries and merge the results manually.